### PR TITLE
style edits

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

File: pages/about.html [hunk 1]
```diff
@@ -4,12 +4,12 @@
 title: About
 ---
 
-<p>Welcome to the alpha site for the U.S. Web Design Standards project. </p>
+<p>Welcome to the alpha site for the U.S. Web Design Standards project.</p>
```
Comment (carodew) [inline pages/about.html]: Think we can delete the word 'project'? Sounds horsey to me.

Comment (awfrancisco) [inline pages/about.html]: Yup.

File: pages/all.html [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
 
-<p>This site is currently under development and your feedback is important to us. We encourage you to contribute your input at <a href="https://github.com/18F/usfwds/issues">https://github.com/18F/usfwds/issues</a>.</p> 
+<p>This site is currently under development and your feedback is important to us. We encourage you to contribute your input at <a href="https://github.com/18F/usfwds/issues">https://github.com/18F/usfwds/issues</a>.</p>
 
-<p>The U.S. Federal Web Design Standards project is a partnership of the <a href="https://whitehouse.gov/usds">U.S. Digital Service</a> and <a href="https://18f.gsa.gov">18F</a> with the support of partners across the Federal Government. If you'd like to learn more, please send a note to <a href="mailto:govtwidepatternlibrary@gsa.gov">govtwidepatternlibrary@gsa.gov</a>.</p>
+<p>The U.S. Web Design Standards project is a partnership of the <a href="https://whitehouse.gov/usds">U.S. Digital Service</a> and <a href="https://18f.gsa.gov">18F</a> with the support of partners across the government. If you'd like to learn more, please send a note to <a href="mailto:govtwidepatternlibrary@gsa.gov">govtwidepatternlibrary@gsa.gov</a>.</p>
```
Comment (carodew) [inline pages/all.html]: Dang. I missed this one. Thanks for catching it!

**Comment by carodew** [comment]
@awfrancisco I made a couple extra tweaks that would be great for you to look at. 
@mbland and in the process created another merge conflict. :cry:

**Comment by mbland** [comment]
No problem. Fixed. Gotta love trailing whitespace conflicts.

**Comment by awfrancisco** [comment]
Looks all good to me!

**Pull request merged**